geomstats/geometry/beta_distributions.py

```diff
@@ -159,20 +162,18 @@ def inner_product_matrix(self, base_point=None):
         -------
         base_point : array-like, shape=[n_samples, 2, 2]
         """
-        if base_point is not None:
+        if base_point is None:
             raise ValueError('The metric depends on the base point.')
```

Comment:
We should start unit testing the errors: it would be an "easy" code coverage improvement, e.g.:
https://github.com/geomstats/geomstats/blob/1e33eb3841cda03348a9640e247f86a244edaf75/tests/test_manifold.py#L20

I can help on this today, let's just make sure we don't address the same ones.

Reply:
Sure, I'll do some of these in the coming days. I'll start by the end (alphabetical order)

Reply:
btw using a lambda function as in the example you pointed out raises a DeepSource antipattern issue... Should we keep it or define a function to do it?